compiler/plc_diagnostics/src/diagnostics/error_codes/E114.md

```diff
@@ -0,0 +1,10 @@
+# Cannot extend a POU multiple times 
+
+Multiple inheritance is not allowed. POUs can only be derived from a single parent POU.
```

Comment:
```suggestion
Multiple `EXTENDS` keywords are not allowed
```
We will eventually have mutliple inheritance on interfaces, and we can add an extra validation for POUs. this is only for the `EXTENDS` keyword now